cw-orch-daemon/README.md

```diff
@@ -0,0 +1 @@
+../README.md
```

Comment:
We should include some details as to what the package is, or at least link it dynamically with md `See [top-level README] (../README.md)` (with no space)